planning/planning_error_monitor/launch/planning_error_monitor.launch.xml

```diff
@@ -0,0 +1,17 @@
+<launch>
+  <!--final output-->
+  <arg name="input/trajectory" default="/planning/scenario_planning/trajectory" />
+  <!--avoidance-->
+  <!--arg name="input/trajectory" default="/planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/trajectory" /-->
+  <!--surround obstacle check-->
+  <!--arg name="input/trajectory" default="/planning/scenario_planning/lane_driving/motion_planning/surround_obstacle_checker/trajectory" /-->
+  <!--obstacle stop-->
+  <!--arg name="input/trajectory" default="/planning/scenario_planning/lane_driving/trajectory" /-->
```

Comment:
Could you remove or turn off the comment o　ut?

Reply:
@1222-takeshi 
this is the error detection package and this node analyze different layer of trajectory so I added  with comment out in order to refer each trajectory easily.

@kenji-miyake 
Do you have any good idea to have optional topics as input?

Reply:
Hmm, I think you can add an option `trajectory_type` or so.

```
// pseudo code
if trajectory_type == scenario_planning
    /planning/scenario_planning/trajectory
else if trajectory_type == avoidance
    /planning/scenario_planning/lane_driving/motion_planning/obstacle_avoidance_planner/trajectory
else if ...
end
```

Reply:
Or just adding some comments would be fine, I feel.

Reply:
@kenji-miyake 
is it possible in .xml files to have condition?

Reply:
Like this.
https://github.com/tier4/AutowareArchitectureProposal_launcher/blob/2af084219d0fae7effa46ef1277bb8f2082f19c6/system_launch/launch/system.launch.xml

Reply:
@1222-takeshi @kenji-miyake 
fixed like this 99bb00b